tools/llm_bench/llm_bench_utils/ov_utils.py

```diff
@@ -436,15 +429,16 @@ def get_vae_decoder_infer_duration(self):
         def raw_metrics(self):
             return RawImGenPerfMetrics(self.iteration_time)
 
+    image_gen_use_case = kwargs['use_case']
     image_gen_pipeline_class = openvino_genai.Text2ImagePipeline
-    if (kwargs.get("task") == TASK["inpainting"] or ((kwargs.get("media") or kwargs.get("images")) and kwargs.get("mask_image"))
+    if (kwargs.get("task") == image_gen_use_case.TASK["inpainting"]["name"] or ((kwargs.get("media") or kwargs.get("images")) and kwargs.get("mask_image"))
             or "Inpaint" in model_index_data.get("_class_name", "")):
```

Comment:
Naming some parts of the condition will be beneficial for readability.

Reply:
moved conditions to separate functions